src/lib.rs

```diff
@@ -119,21 +126,41 @@ pub fn jaro(a: &str, b: &str) -> f64 {
     }
 }
 
-/// Like Jaro but gives a boost to strings that have a common prefix.
+struct StringWrapper<'a>(&'a str);
+
+impl<'a, 'b> IntoIterator for &'a StringWrapper<'b> {
```

Comment:
Does this approach cause any sort of performance hit to using the string functions?

Reply:
No, this wrapper should disappear at compilation.